### PR TITLE
PT-158864866: throw away keys

### PR DESCRIPTION
**Pull request by fabiankrol**

### CODE REVIEW

**Comment by fabiankrol** [comment]
Endpoint for fetching beneficiary key should be introduce to improve readability. Related PT: https://www.pivotaltracker.com/n/projects/2124891/stories/161532247

File: epoch/api/oracle_api_usage.md [hunk 1]
```diff
@@ -16,12 +16,18 @@ The following assumes that the node exposes at address `localhost/127.0.0.1` the
 
 ## Oracle management flow
 
-In order to work through the example we also need the (Base58Check-encoded)
-public key of the node. This is easily retrieved from the running node:
+In order to work through the example we need the (Base58Check-encoded)
+public key of the account with positive balance. For simplicity, we will use
+beneficiary account. Public key can be found in configuration file
+( `mining > beneficiary` parameter).
```
Comment (zp-sd) [inline epoch/api/oracle_api_usage.md]: not sure if whitespace before `mining` is necessary

File: epoch/api/naming_system_api_usage.md [hunk 2]
```diff
@@ -12,12 +12,23 @@ The following assumes that the node exposes at address 127.0.0.1 the following p
 
 ## Name management flow
 
-Check the public key of the node first (used as account key in later API calls):
+In order to work through the example we need the (Base58Check-encoded)
+public key of the account with positive balance. For simplicity, we will use
+beneficiary account. Public key can be found in configuration file
+( `mining > beneficiary` parameter).
```
Comment (zp-sd) [inline epoch/api/naming_system_api_usage.md]: not sure if whitespace before `mining` is necessary

File: epoch/api/oracle_api_usage.md (hunk 1 above)
Comment (fabiankrol) [reply epoch/api/oracle_api_usage.md]: It's not ;)

**Comment by fabiankrol** [comment]
Ref: `Introduce debug api for beneficiary pubkey` https://github.com/aeternity/epoch/pull/1712

**Pull request merged**